include/verilated_vpi.cpp

```diff
@@ -263,33 +265,24 @@ class VerilatedVpioScope VL_NOT_FINAL : public VerilatedVpio {
 protected:
     const VerilatedScope* const m_scopep;
     bool m_toplevel = false;
+    std::string m_name;
+    std::string m_fullname;
```

Comment:
I think as the only thing the logic above this class does is trim the front off from compile-time constant strings, these could be char*'s which would be faster.